clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTaskTest.java

```diff
@@ -2004,14 +2003,14 @@ public void testCompleteCalledWhenUnsubscribeAfterBatchPushDisabled(AAConfig aaC
 
     runTest(Utils.setOf(PARTITION_FOO), () -> {
       Store mockStore = mock(Store.class);
-      storeIngestionTaskUnderTest.unSubscribePartition(fooTopicPartition);
```

Comment:
If we set "SERVER_PROCESS_CONSUMER_ACTION_WITHOUT_ENQUEUE_ENABLE" to false and keep the unsubscribe action here, will the test pass? Just want to double check, we can make this test work for the new config.